src/camera_display.cpp

```diff
@@ -374,6 +379,50 @@ void CameraPub::updateFrameRate()
 {
 }
 
+void CameraPub::updateDisplayName()
+{
+    std::string name = name_property_->getStdString();
+    if (name.empty())
+    {
+        setStatus(StatusProperty::Warn, "Display name",
+                  "Should not be empty.");
+        return;
+    }
+
+    ros::NodeHandle nh_private_ = ros::NodeHandle("~"+name);
```

Comment:
In this case, we should rename it to "display namespace" so that's more obvious that namespace rules apply.